mri_team_count/poll_repo.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
This code is for detecting any changes in the `mri_team_count` of the github repo to run the `create_mri_team_count_series_view` & `create_mri_team_count_view` query.